tests/std/tests/P0896R4_ranges_range_machinery/test.cpp

```diff
@@ -1609,6 +1609,22 @@ struct strange_view5 : strange_view4, ranges::view_interface<strange_view5> {
 STATIC_ASSERT(!std::is_base_of_v<std::ranges::view_base, ranges::view_interface<strange_view4>>);
 STATIC_ASSERT(!std::is_base_of_v<std::ranges::view_base, ranges::view_interface<strange_view5>>);
 
+// Verify that enable_view<T&> or enable_view<T&&> is never true
+STATIC_ASSERT(!ranges::enable_view<strange_view4&>);

```

Comment:
We should begin by verifying that `ranges::enable_view<strange_view4>` is true. (In general, tests should verify at least one true and one false scenario - this prevents damaged inputs or test machinery from always being true/false and silently passing.)